doc/source/whatsnew/v0.24.0.txt

```diff
@@ -17,6 +17,8 @@ New features
 
 - ``ExcelWriter`` now accepts ``mode`` as a keyword argument, enabling append to existing workbooks when using the ``openpyxl`` engine (:issue:`3441`)
 
+- :func:`DataFrame.to_parquet` now accepts ``index`` as an argument, allowing the user to override the engine's default behavior and include or omit the dataframe's indexes from the resulting Parquet file. (:issue:`20768`)
```

Comment:
say the default is to *include* the index

Reply:
> say the default is to _include_ the index

Not necessarily if you're using the `fastparquet` engine. According to the [docs](https://fastparquet.readthedocs.io/en/latest/api.html#fastparquet.write), the index is _only_ written if it's not an integer sequence 0..._n_.

Reply:
The default depends on the engine (that's the reason it is None by default, to answer your other question). See the discussion above, eg https://github.com/pandas-dev/pandas/pull/22266#pullrequestreview-146069083. 

The default of None is certainly not the only possibility. As noted above, we can also make it `index=True` by default to have consistency for both engines. But, this would mean a change in behaviour for fastparquet engine. 
On the one hand, I certainly favor this consistency, but on the other hand, I think the default of fastparquet actually makes sense ..

Reply:
> I think the default of fastparquet actually makes sense

Agreed. It doesn't really add information to the archive since the order of the rows is already known.